src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -455,16 +464,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		return errors.Wrapf(err, "Timeout while waiting for console to become available")
 	}
 
+	// Wait for OLM operators
+	if err = c.waitForOLMOperators(ctx); err != nil {
+		return errors.Wrapf(err, "Error while initializing OLM operators")
+	}
+
+	return nil
+}
+
+func (c controller) waitForOLMOperators(ctx context.Context) error {
+	// Get maximum wait timeout for OLM operators:
+	operators, err := c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID)
```

Comment:
I think this part must be retry-able no?

Reply:
Done.